Makefile.nt

```diff
@@ -18,8 +18,62 @@ include Makefile.shared
 defaultentry:
 	@echo "Please refer to the installation instructions in file README.win32.adoc."
 
+REQUIRE_FLEXDLL:=$(if $(shell which flexlink 2>/dev/null),,$(FLEXDLL_SRC)/flexlink$(EXE))
+BOOT_FLEXLINK_CMD=$(if $(REQUIRE_FLEXDLL),FLEXLINK_CMD="../boot/ocamlrun ../$(FLEXDLL_SRC)/flexlink.exe")
+# Sys.command doesn't accept / in PATHs (because cmd doesn't)
```

Comment:
FLEXDLL_SRC started out earlier in the process where the user would be downloading the tarball, so may extract, say, to flexdll-0.34 and choose to override the location instead of renaming the directory. I actually prefer to hard code it :)
It would be better if all Makefile's consistently had ROOTDIR, but there are quite a few which - presumably for legacy reasons - don't, and I was nervous of changing that because that would touch non-Windows Makefile's. I don't think I've added a totally new definition of CAMLOPT anywhere? I was mentally leaving refactoring like that until a grand attempt to merge the build systems completely :smile: